values update q2 2020 (#36)
fishtown-analytics/corp: values.md

We are a mission-driven company. We believe that profits are an important tool in the pursuit of our mission, but they are not the mission itself.

### We contribute to the knowledge loop.

The highest goal of any human is to produce new knowledge that can subsequently be built upon by others. This is the process upon which every good thing in human society has been built. We participate in this most fundamental of human endeavors by thinking in public and defaulting to open source.

### We are more concerned with value creation than value capture.

> We will only succeed if we create value for our community. If we optimize for that first, commercial success will follow.

Reviewer: I think this is more accurate and more general:

```suggestion
We will only succeed if we create value in the world. If we optimize for that first, commercial success will follow.
```

### Users are our best advocates.

We market our products and services by working with amazing humans, making them incredibly happy, and asking them to spread the word. We go above and beyond for our clients / users to invest in our future growth.

### We are humble.

While we believe that we are building something exceptional, that knowledge does not make us feel proud, it makes us feel humble. It imbues us with a deep sense of responsibility towards our clients, coworkers, and the community at large.

### We optimize for the long-term.

While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.

### We work hard and go home.

We take our responsibilities as parents, citizens, individuals, family members, and members of our various communities just as seriously as we take our commitments at work. When we are on, we are focused. When we are off, we are free to be fully disconnected.

### We treat employees like people.

> The only way to build a team of humans who innovate in shared service to a mission is to _treat them like people_. And people only want a few things: economic security, self-determination, health, belonging, and meaning. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness.

Reviewer: I don't love this section -- the verbiage here is too euphemistic and hand-wavy, while also being overly prescriptive about what people _want_. Here's my take, happy to workshop or discuss!

### We are human.

We are human beings innovating in a shared service to a mission. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness. We bring our whole selves to work, and we recognize that our identities extend beyond the work that we do.

Author: this is a fantastic change. I struggled writing this section; yours is much better.
